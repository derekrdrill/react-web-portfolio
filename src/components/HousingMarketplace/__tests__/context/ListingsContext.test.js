import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';

import ListingsProvider from '../../context/ListingsContext';

const renderer = new ShallowRenderer();

describe('ListingsProvider with me tests', () => {
  it('renders correctly', () => {
    renderer.render(<ListingsProvider />);
    const result = renderer.getRenderOutput();
    expect(result).toMatchSnapshot();
  });
});
