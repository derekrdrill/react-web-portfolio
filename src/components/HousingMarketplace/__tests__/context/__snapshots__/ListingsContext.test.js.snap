// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ListingsProvider with me tests renders correctly 1`] = `
<Context.Provider
  value={
    Object {
      "deleteComplete": false,
      "isConfirmingDelete": false,
      "isDeleting": false,
      "isEditing": false,
      "listingsDispatch": [Function],
      "modalLocation": "",
      "modalName": "",
    }
  }
/>
`;
